internal/access/signup.go

```diff
@@ -35,7 +35,12 @@ func SignupEnabled(c *gin.Context) (bool, error) {
 		return false, err
 	}
 
-	return identities+providers+grants == int64(0), nil
+	accessKeys, err := data.Count[models.AccessKey](db.Unscoped())
+	if err != nil {
+		return false, err
+	}
+
+	return identities+providers+grants+accessKeys == int64(0), nil
```

Comment:
Is the `int64` conversion necessary?

Reply:
I thought it is because it was complaining previously but doesn't seem to be the case anymore